common/src/main/java/tachyon/retry/ExponentialBackoffRetry.java

```diff
@@ -29,6 +29,13 @@
   private final int mBaseSleepTimeMs;
   private final int mMaxSleepMs;
 
+  /**
+   * Constructs a new retry which grows exponentially.
```

Comment:
Constructs a new retry facility which sleeps for an exponentially increasing amount of time between retries.